webui/react/src/components/Navigation.tsx

```diff
@@ -33,7 +33,6 @@ const Navigation: React.FC<Props> = ({ children }) => {
   const fetchMyRoles = useFetchUserRolesAndAssignments(canceler);
 
   usePolling(fetchAgents);
-  usePolling(fetchWorkspaces);
```

Comment:
i think we do need to use `usePolling` cuz the new change doesn't periodically fetch new workspaces. if someone adds workspaces on the different computer, this code cannot fetch these until user refresh the page